### PR TITLE
As/update address

### PR DESCRIPTION
**Pull request by soncuteanca**
# Description
This verifies that adding an additional name updates the existing saved address entry.

## Bugzilla bug ID

**ID: 1895877**
**Link: https://bugzilla.mozilla.org/show_bug.cgi?id=1895877**

## Type of change

Please delete options that are not relevant.

- [x ] New Test

# How does this resolve / make progress on that bug?
Completed

Please describe the progress or significance with respect to the bug listed above.

# Screenshots / Explanations

Please upload any relevant media or add a relevant description with respect to the bug listed above.

# Comments / Concerns
There are many ways to trigger the Update doorhanger, but there are still some inconsistencies that need more attention from the engineering side. Adding an additional name is the standard one used in our manual testing, that should work every time.

Please add a short blurb about any comments or concerns that this change might cause.

### CODE REVIEW

File: modules/data/about_prefs.components.json [hunk 1]
```diff
@@ -8,7 +8,6 @@
             "searchEngineDropdown"
         ]
     },
-
```
Comment (ben-c-at-moz) [inline modules/data/about_prefs.components.json:11]: Were these line deletions done automatically by PyCharm? I kind of prefer the blank lines for readability, but it's not a dealbreaker.

**Review by ben-c-at-moz — APPROVED** [review]
One minor issue, but the code looks good!

File: modules/data/about_prefs.components.json (hunk 1 above)
Comment (soncuteanca) [reply modules/data/about_prefs.components.json]: On reformat file (Ctrl + Alt + L ) is the pycharm doing. I added the spaces again to match the rest of the json files

File: tests/form_autofill/test_updating_address.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import time
```
Comment (ben-c-at-moz) [inline tests/form_autofill/test_updating_address.py:1]: Are you using `time` somewhere in here? I don't see it.

**Review by ben-c-at-moz — APPROVED** [review]
One minor thing, would love for you to look at that before merging.

File: tests/form_autofill/test_updating_address.py (hunk 2 above)
Comment (soncuteanca) [reply tests/form_autofill/test_updating_address.py]: I did, while creating the test, time.sleep, to help me with visualization. I just forget to remove it. I did another commit to remove the import.

**Review by ben-c-at-moz — APPROVED** [review]
good to go!

**Pull request merged**